src/DI/ContainerBuilder.php

```diff
@@ -266,7 +266,9 @@ public function autowireArguments($class, $method, array $arguments)
 		if (!$rm->isPublic()) {
 			throw new ServiceCreationException("$class::$method() is not callable.");
 		}
-		$this->addDependency($rm->getFileName());
+		if (!$rm->isInternal()) {
```

Comment:
`addDependency` is used 3× times (here too https://github.com/nette/di/blob/ea244d0662c164a164bb91e622b00dc361f0668c/src/DI/ContainerBuilder.php#L696), so what about `$this->addDependency((string) $rm->getFileName())` and keep it simple?